tests/models/maskformer/test_feature_extraction_maskformer.py

```diff
@@ -345,6 +347,174 @@ def common(is_instance_map=False, segmentation_type=None):
         common(is_instance_map=False, segmentation_type="pil")
         common(is_instance_map=True, segmentation_type="pil")
 
+    @slow
```

Comment:
Why is this marked as slow? How long does it take to run? The other times can be found [here](https://output.circle-artifacts.com/output/job/e9972e7f-4d1b-410f-be39-8f7dcf300336/artifacts/0/~/transformers/reports/tests_torch/durations.txt)